docs/content/manpages/geowave-dbscan.txt

```diff
@@ -0,0 +1,144 @@
+//:geowave-dbscan(1)
+//:======================
+//::doctype: manpage
+
+NAME
+//:----
+
+geowave analytic dbscan - Density Based Scanner
+
+SYNOPSIS
+//:--------
+
+*geowave analytic dbscan* [options] <storename>
+
+DESCRIPTION
+//:-----------
+
+The geowave analytic dbscan operator will run a density based scanner analytic on GeoWave data
+
+OPTIONS
+//:-------
+
+-dt, --clusteringDistanceThresholds::
+
+   Comma separated list of distance thresholds, per dimension
+
+-du, --clusteringGeometricDistanceUnit::
+
+   Geometric distance unit (m=meters,km=kilometers, see symbols for
+   javax.units.BaseUnit)
+
+-cmi, --clusteringMaxIterations::
+
+   Maximum number of iterations when finding optimal clusters
+
+-cms, --clusteringMinimumSize::
+
+   Minimum Cluster Size
+
+-cdf, --commonDistanceFunctionClass::
+
+   Distance Function Class implements
+   mil.nga.giat.geowave.analytics.distance.DistanceFn
+
+-eit, --extractAdapterId::
+
+   Input Data Type ID
+
+-ei, --extractIndexId::
```

Comment:
The index identifier of the index to query.  If not provided, one will be chosen.